_articles/getting-paid.md

```diff
@@ -111,7 +111,7 @@ Depending on your personal circumstances, you can try raising money independentl
 
 Finally, sometimes open source projects put bounties on issues that you might consider helping with. 
 
-* @ConnorChristie was able to get paid for [helping](https://github.com/MARKETProtocol/MARKET.js/issues/14) @MARKETProtocol work on their javascript library [through a bounty on gitcoin](https://gitcoin.co/).
+* @ConnorChristie was able to get paid for [helping](https://web.archive.org/web/20181030123412/https://webcache.googleusercontent.com/search?strip=1&q=cache:https%3A%2F%2Fgithub.com%2FMARKETProtocol%2FMARKET.js%2Fissues%2F14) @MARKETProtocol work on their javascript library [through a bounty on gitcoin](https://gitcoin.co/).
```

Comment:
Can't believe I've never seen this (archived copy of Google cache where archive doesn't have bare URL) before. Nice!

Reply:
Inspiration in desperation :-D